src/project-tests/survey-form-tests.js

```diff
@@ -15,58 +13,81 @@ export default function createSurveyFormTests() {
         return numActive;
     }
 
+    /**
+    Returns the number of elements that have a value-attribute
+    **/
+    function getNumValues(elems) {
+        var numValues = 0;
+        for (var i = 0; i < elems.length; i++) {
+            var el = elems[i];
+            if (el.hasAttribute("value")) {
+                numValues++;
+            }
+        }
+        return numValues;
+    }
+    
     describe("Survey Form tests", function() {
         describe("#Content", function() {
 
             it('1. I can see a title with id="title" in H1 sized text.', function() {
                 var title = document.getElementById('title');
-                FCC_Global.assert.isNotNull(title, "There should be an element with id='title '");
+                FCC_Global.assert.isNotNull(title, 'There should be an element with id="title" ');
                 FCC_Global.assert.strictEqual(title.tagName, "H1", "#title should be in H1 sized text ");
-                FCC_Global.assert.isAbove(title.innerText.length, 0, '#title should contain some text ')
+                FCC_Global.assert.isAbove(title.innerText.length, 0, '#title should contain some text ');
             });
 
             it('2. I can see a short explanation with id="description" in P sized text.', function() {
                 var description = document.getElementById('description');
-                FCC_Global.assert.isNotNull(description, "There should be an element with id='description' ");
+                FCC_Global.assert.isNotNull(description, 'There should be an element with id="description" ');
                 FCC_Global.assert.strictEqual(description.tagName, "P", "#description should be in P sized text ");
                 FCC_Global.assert.isAbove(description.innerText.length, 0, '#description should contain some text ')
             });
+            
+            it('3. I am presented with a <form> with id="survey-form". The <form> must contain all the form widgets.', function() {
+                var form = document.getElementById("survey-form");
+                FCC_Global.assert.isNotNull(form, 'There should be an element with id="survey-form" ');
+                FCC_Global.assert.strictEqual(form.tagName, "FORM", "#survey-form should be a <form>-element ");
+            });
 
```

Comment:
My only concern with this one is the wording... not sure about the usage of the word "widgets". Can we go with something more generic like "elements" instead?